diff --git a/node_modules/expo-modules-core/package.json.bak b/node_modules/expo-modules-core/package.json.bak
new file mode 100644
index 0000000..ee38ddc
--- /dev/null
+++ b/node_modules/expo-modules-core/package.json.bak
@@ -0,0 +1,48 @@
+{
+  "name": "expo-modules-core",
+  "version": "2.0.0",
+  "description": "The core of Expo Modules architecture",
+  "main": "index.js",
+  "types": "build/index.d.ts",
+  "sideEffects": [
+    "*.fx.ts"
+  ],
+  "scripts": {
+    "build": "expo-module build",
+    "clean": "expo-module clean",
+    "lint": "expo-module lint",
+    "test": "expo-module test",
+    "prepare": "expo-module prepare",
+    "prepublishOnly": "expo-module prepublishOnly",
+    "expo-module": "expo-module"
+  },
+  "keywords": [
+    "expo",
+    "modules",
+    "expo-modules",
+    "unimodules",
+    "react-native"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/expo/expo.git",
+    "directory": "packages/expo-modules-core"
+  },
+  "bugs": {
+    "url": "https://github.com/expo/expo/issues"
+  },
+  "author": "650 Industries, Inc.",
+  "license": "MIT",
+  "homepage": "https://github.com/expo/expo/tree/main/packages/expo-modules-core",
+  "jest": {
+    "preset": "expo-module-scripts"
+  },
+  "dependencies": {
+    "invariant": "^2.2.4"
+  },
+  "devDependencies": {
+    "@testing-library/react-native": "^12.5.2",
+    "expo-module-scripts": "^4.0.0"
+  },
+  "gitHead": "1f84b4c952ed9785c41ee65c9877e44f1df27af9"
+}
diff --git a/node_modules/expo-modules-core/package.json.patch b/node_modules/expo-modules-core/package.json.patch
new file mode 100644
index 0000000..ad607d5
--- /dev/null
+++ b/node_modules/expo-modules-core/package.json.patch
@@ -0,0 +1,5 @@
+--- package.json.orig
++++ package.json
+@@ -1 +1 @@
+-  "main": "src/index.ts",
++  "main": "build/index.js",
